src/ResourceManager/MachineLearning/Commands.MachineLearning/Utilities/WebServicesClient.cs

```diff
@@ -0,0 +1,142 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components;
+using Microsoft.Azure.Management.MachineLearning.WebServices;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Models;
+using APIClient = Microsoft.Azure.Management.MachineLearning.WebServices.AzureMLWebServicesManagementClient;
+
+namespace Microsoft.Azure.Commands.MachineLearning.Utilities
+{
+    public class WebServicesClient
+    {
+        private const int AsyncOperationPollingIntervalSeconds = 5;
+
+        private readonly APIClient apiClient;
+
+        public WebServicesClient(AzureContext context)
+        {
+            this.apiClient = AzureSession.ClientFactory.CreateArmClient<APIClient>(context, AzureEnvironment.Endpoint.ResourceManager);
+            this.apiClient.LongRunningOperationRetryTimeout = WebServicesClient.AsyncOperationPollingIntervalSeconds;
```

Comment:
May I know how could I set the interval only for playback testing? Since it is a retry timeout, I think it is fine for playback test because there will no need to retry in the local playback.